packages/client/src/pages/SignIn/index.tsx

```diff
@@ -1,3 +1,83 @@
+import { Link } from 'react-router-dom';
+import { useActions } from '../../hooks/useActions';
+import { useTypeSelector } from '../../hooks/useTypeSelector';
+import { RouterList } from '../../router/routerList';
+import {
+  useLazyCheckAuthUserQuery,
+  useLogoutMutation,
+  useSignInMutation,
+} from '../../store/auth/api';
+import { useLazyFetchUserDataQuery } from '../../store/user/api';
+import type { RequestSignIn } from '../../store/auth/type';
+import type { RequestUserData } from '../../store/user/type';
+
 export default function SignIn() {
-  return <>SignIn</>;
+  const { isAuth } = useTypeSelector(state => state.auth);
+  const { userData } = useTypeSelector(state => state.user);
+
+  const [fetchAuthUser] = useLazyCheckAuthUserQuery();
+  const [fetchUserData] = useLazyFetchUserDataQuery();
+  const [fetchLogout] = useLogoutMutation();
+  const [fetchSignIn] = useSignInMutation();
+
+  const { setIsAuth, resetUserState, setUserData } = useActions();
+
+  const handleCheckIsAuth = () => {
+    fetchAuthUser(null).then(res => {
+      setIsAuth(res.isSuccess);
```

Comment:
Здесь можно деструктурировать
```suggestion
    fetchAuthUser(null).then(({ isSuccess }) => {
      setIsAuth(isSuccess);
```

Reply:
+